mlir/test/Target/LLVMIR/arm-neon.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file  %s | FileCheck %s
```

Comment:
If you're adding `-split-input-file`, could you make sure that all cases are separated with `// -----`? Similar suggestion for the other test files.